lib/route_translator/translator/path/segment.rb

```diff
@@ -8,14 +8,10 @@ def translate_string(str, locale, scope)
             opts = { scope: scope, locale: locale }
             res = I18n.translate(str, opts)
```

Comment:
Fallback options change the behaviour of I18n. 
I18n Cascade backend looked like a good fit here, but it stops lookups even when a Hash is returned.